src/main/java/uk/gov/pay/adminusers/service/EmailService.java

```diff
@@ -34,7 +34,7 @@ public EmailService(NotificationService notificationService, ServiceDao serviceD
     }
 
     private String formatMerchantAddress(MerchantDetailsEntity merchantDetails) {
-        StringJoiner merchantAddress = new StringJoiner(", ", "","");
+        StringJoiner merchantAddress = new StringJoiner(", ", "", "");
```

Comment:
thank you for making the whitespace uniform 🥇

Reply:
@simad the power of `Cmd + Alt + L` :D